app/payload_mapper.py

```
@@ -0,0 +1,84 @@
+    tof_error_text = "" if tof_error is None else str(tof_error)
+
+    return (
+        Point("sensor_raw")
+        .tag("device_id", str(data.get("deviceId", "UNKNOWN_DEV")))
+        .tag("source", str(data.get("source", "UNKNOWN_SRC")))
+        .field("seq", _safe_int(data.get("seq")))
+        .field("sample_interval_ms", _safe_int(data.get("sampleIntervalMs"), 100))
+        .field("pir_motion", _safe_bool(sensors.get("pirMotion")))
+        .field("pir_value", _safe_int(sensors.get("pirValue")))
+        .field("tof_distance_mm", _safe_int(sensors.get("tofDistanceMm")))
+        .field("tof_valid", _safe_bool(sensors.get("tofValid"), True))
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)